### DISCUSSION TITLE
Neovim search + flash.nvim integration?

### DISCUSSION THREAD

**Discussion by Demianeen** [LazyVim/LazyVim#3345] [Q&A] [answered]
There used to be search feature that was similar to flash.nvim where you can write symbol to instantly jump to the next search match. But I can't see this feature anymore. Was it removed or moved to extras?
![CleanShot 2024-05-28 at 10 14 04@2x](https://github.com/LazyVim/LazyVim/assets/51330172/fbe19772-ed06-4ba7-8b66-5f42335f910a)

**Comment by dpetka2001** [comment]
I think you're referring to [Leap Extra](https://www.lazyvim.org/extras/editor/leap)? It was moved to Extras ever since `flash.nvim` became default.

**Reply by folke** [reply]
He probably means the flash search integration

**Reply by dpetka2001** [reply]
Oh, the feature that was disabled by default compared to how it was in the past. I thought he meant plugin, that's why I suggested leap. My bad.

**Reply by dpetka2001** [reply, marked as answer]
In that case @Demianeen take a look [here](https://github.com/folke/flash.nvim/blob/11a2e667d19d8f48c93c6ed2e2e525ac6b1b79be/README.md?plain=1#L236). You need to change this in your personal configuration to `true`.
